src/components/GlobalStyles.tsx

```diff
@@ -1,20 +1,25 @@
 import { css } from '@emotion/core'
-import { colors, fonts, getCdnFontFaces } from '@hedviginsurance/brand'
+import {
+  colors,
+  colorsV3,
+  fonts,
+  getCdnFontFaces,
+} from '@hedviginsurance/brand'
 
-export const globalStyles = () => css`
+export const globalStyles = (brandPivot: boolean) => css`
```

Comment:
Rather than making a flag out of this, could we have separate `globalStylesBrandPivot`'? so we do an extension of the `globalStyles` for brand pivot. If possible i think we should try to avoid bool flags like this, looking at where the function is used its quite hard to make out what `true` or `false` means